CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.2.0] - 2023-06-30
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.1.1...v1.2.0)
+
+### Added
+* Optimizations for [ListObjects](https://openfga.dev/api/service#/Relationship%20Queries/ListObjects) and [StreamedListObjects](https://openfga.dev/api/service#/Relationship%20Queries/StreamedListObjects) for models involving intersection (`and`) and exclusion (`but not`) (#797)
+
+### Changed
```

Comment:
can you add https://github.com/openfga/openfga/pull/829

i'd like people that are using this sample to re-download it